src/content/migrate/4.md

```diff
@@ -11,23 +11,30 @@ This guide only shows major changes that affect end users. For more details plea
 
 ## Node.js v4
 
-If you are still using Node.js v4 or lower, you need to upgrade your Node.js installation to Node.js v6 or higher.
+*TLDR:; Node.js versions lower than 6 are no longer supported. So, upgrade to the latest Node.js LTS verison.*
 
+If you are still using Node.js v4 or lower, you need to upgrade your Node.js installation to Node.js v6 or higher. Ideally you should be getting the latest Long Term Support(LTS) version of Node.js. Instructions for upgrading your Node.js version can be found [here](https://stackoverflow.com/questions/10075990/upgrading-node-js-to-latest-version). This is because the source of webpack was changed to a higher ECMAScript version in webpack 4.
 
 ## CLI
 
 The CLI has moved to a separate package: webpack-cli. You need to install it before using webpack, see [basic setup](/guides/getting-started/#basic-setup).
 
+The installation guide can be found [here](/src/content/guides/installation.md).
+
 
 ## Update plugins
 
-Many 3rd-party plugins need to be upgraded to their latest version to be compatible.
+Many 3rd party plugins need to be updated to the latest versions to be compatible with webpack 4. Many of the popular plugins and the links to their repositories can be found [here](https://github.com/webpack-contrib/awesome-webpack#webpack-plugins).
+
+A more informative resource can be found [here](https://medium.com/webpack/webpack-4-migration-guide-for-plugins-loaders-20a79b927202).
 
 
 ## mode
 
 Add the new [`mode`](/concepts/mode/) option to your config. Set it to production or development in your configuration depending on config type.
 
+More information on the different types of `mode`'s like `none`, `development`, and `production` are mentioned in [here](https://webpack.js.org/concepts/mode/#usage).
```

Comment:
I think that this addition is redundant, previous paragraph has the same page linked

Reply:
Removed it.